<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智慧农业助手</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/icon/ny.jpg);</string>
  </property>
  <widget class="QPushButton" name="BtnOpen">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(0, 0, 0);
	background-image: url(:/icon/R-C.jfif);
	border: none;
	border-radius: 10px;
	color: rgb(255, 255, 255);
    background-color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgb(122, 211, 140), stop:1 rgb(59, 195, 129));
}

QPushButton:pressed {
	border: 2px solid qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgb(122, 211, 140), stop:1 rgb(59, 195, 129));
	color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgb(122, 211, 140), stop:1 rgb(59, 195, 129));
	background-color: rgb(255, 255, 255);
}

QPushButton:disabled {
	border: 2px solid qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.5, stop:0 rgb(122, 211, 140), stop:1 rgb(59, 195, 129));
	color: rgb(153, 153, 153);
	background-color: rgb(212, 228, 235);
}</string>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_motor">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
font: 75 italic 12pt &quot;Bahnschrift&quot;;
color: rgb(255, 255, 0);
</string>
   </property>
   <property name="text">
    <string>马达</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_light">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(255, 255, 0);
font: 75 italic 12pt &quot;Bahnschrift&quot;;
</string>
   </property>
   <property name="text">
    <string>灯</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffff00;&quot;&gt;串口号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CBoxSerialPort">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
      <item>
       <widget class="QLabel" name="label_tem">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; color:#00ff00;&quot;&gt;温度&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_tem">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;
font: 16pt &quot;Arial&quot;;
font: 14pt &quot;Arial&quot;;
color: rgb(255, 0, 0);
selection-background-color: rgb(255, 255, 255);
selection-color: rgb(255, 0, 0);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
      <item>
       <widget class="QLabel" name="label_hum">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; color:#55ff00;&quot;&gt;湿度&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_hum">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;
font: 16pt &quot;Arial&quot;;
color: rgb(255, 0, 0);
selection-background-color: rgb(255, 255, 255);
selection-color: rgb(255, 0, 0);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
      <item>
       <widget class="QLabel" name="label_lum">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600; color:#55ff00;&quot;&gt;光照&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_lum">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;
font: 16pt &quot;Arial&quot;;
color: rgb(255, 0, 0);
selection-background-color: rgb(255, 255, 255);
selection-color: rgb(255, 0, 0);</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>167</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffff00;&quot;&gt;波特率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CBoxBaudrate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Button_light">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>440</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
image: url(:/icon/R-C (1).jfif);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_motor">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>440</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
image: url(:/icon/R-C (1).jfif);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
